src/components/AddStudent.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+
+// properties addStudent is required, function called when Add clicked.
+class AddStudent extends Component {
+      constructor(props) {
+      super(props);
+      this.state = {
+        open: false, 
+        student:{email: "", name: ""} 
+      };
+
+      this.handleInputChange = this.handleInputChange.bind(this);
+    };
+
+    
+    handleClickOpen = () => {
+      this.setState( {open:true} );
+    };
+
+    handleClose = () => {
+      this.setState( {open:false} );
+    };
+
+    handleInputChange = (event) => {
+      event.preventDefault();
```

Comment:
What default event are we preventing here? I think it should be fine to let the event default go as normal here since it's just an input changing.